### ISSUE TITLE
iosxr_policy_map

### ISSUE THREAD

**Issue by rjoensen** [CiscoDevNet/terraform-provider-iosxr#139]
Hello,

iosxr_policy_map is missing some features, and is unable to setup conform-action and exceeded-actions. Also can't seem to be able to setup police rate "percent". 

Hopefully this comes in future releases.

Best wishes,
rjoensen

**Comment by danischm** [comment]
Could you share the equivalent CLI commands you are looking for?

**Comment by rjoensen** [comment]
Hello,

Example is here;

policy-map INGRESS-POLICE-39-PERCENT
 class class-default
  police rate percent 39
   conform-action transmit
   exceed-action drop
  !
 !
 end-policy-map

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-iosxr/commit/441f08f5810d23a34f984d061b38a082eb316b70

**Comment by danischm** [comment]
tested with the following config:

```
resource "iosxr_policy_map_qos" "example" {
  policy_map_name = "INGRESS-POLICE-39-PERCENT"
  classes = [
    {
      name                           = "class-default"
      type                           = "qos"
      police_rate_unit               = "percent"
      police_rate_value              = "39"
      police_conform_action_transmit = true
      police_exceed_action_drop      = true
    }
  ]
}
```

**Comment by danischm** [comment]
Available in v0.2.2
